R/list-transpose.R

```diff
@@ -0,0 +1,98 @@
+#' Transpose a list
+#'
+#' @description
+#' `list_transpose()` turns a list-of-lists "inside-out"; it turns a pair of
```

Comment:
To avoid implying this only works with pairs:

```suggestion
#' `list_transpose()` turns a list-of-lists "inside-out". For instance it turns a pair of
```